src/SprykerEco/Shared/UnzerApi/UnzerApiConstants.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Shared\UnzerApi;
+
+/**
+ * Declares global environment configuration keys. Do not use it for other class constants.
+ */
+interface UnzerApiConstants
+{
+    /**
+     * @var string
+     */
+    public const UNZER_PRIVATE_KEY = 'UNZERAPI:UNZER_PRIVATE_KEY';
+    /**
```

Comment:
```suggestion
    public const UNZER_PRIVATE_KEY = 'UNZERAPI:UNZER_PRIVATE_KEY';

    /**
```

I know it's not mandatory, but empty line would be great ;)